stratum/hal/lib/common/common.proto

```diff
@@ -1053,7 +1043,7 @@ message OpticalChannelInfo {
     uint64 min_time = 7;
   }
 
-  // Optical channel frequency in Hz.
+  // Optical channel frequency in Mega Hz.
```

Comment:
I think this should be Hz, like in common.proto

OpenConfig specifies MHz, TAI is Hz. I'd say Hz in uint64 and convert as needed.